code-challenges.js

```diff
@@ -21,15 +39,45 @@ var num3 = -7
 var randomNouns = ["streetlamp", "potato", "teeth", "conclusion", "nephew", "temperature", "database"]
 // Expected output: ["Streetlamp", "Potato", "Teeth", "Conclusion", "Nephew", "Temperature", "Database"]
 
+const capitalize = (array) =>{
+  // for(let i = 0; array.length; i++){
+  //   array.map(value=>{
+  //     return array.toUpperCase
+  //   })
+//must use a .map to iterate through the array and .touppercase to make the change within
+  let array1 = array.map(value => {
+    //important to put brackets with the 0 to only capitalize one of the ltters. That is indicating to capitalize only the letter in positoin idex 0.
+    //if i were to put no brackets would caplizlatie all the characters in the array
+    return value[0].toUpperCase() + value.substring(1)
+//looked into the syllabus and got the substring  method but dont understand what it does.
+//prior to putting it there it would just give me back the fitst letter capitlaized
 
+  })
+  return array1.join(" ")
```

Comment:
So the expected output, requires this to be an array, there is no need to .join it. Be sure to double check the output requirements! 

> Create a function that takes in the randomNouns variable and **returns an array** with all the words capitalized.